### ISSUE TITLE
Null pointer dereference in sys.h

### ISSUE THREAD

**Issue by ghost** [just-js/modules#2]
You may have a `Null pointer dereference` issue in a `sys.h` file on line 55 :)

https://github.com/just-js/modules/blob/b25c0ee3c3f4b945bfaf2350679a411970a37737/sys/sys.h#L52-L57

I could create a PR if you want

**Comment by billywhizz** [comment]
yes. the code needs a good cleanup. a lot of it is at PoC stage at moment and a number of things i need to review/fix and refactor. i'll fix this in the next relase.

**Comment by ghost** [comment]
@billywhizz I'd like to help/contribute somehow if it's possible :)
